evergreen.yml

```diff
@@ -996,24 +1009,24 @@ functions:
         exit $EXITCODE
 
   "upload start_adf logs":
-     - command: s3.put
-       params:
-         aws_key: ${aws_key}
-         aws_secret: ${aws_secret}
-         local_file: mongosql-odbc-driver/local_adf/logs/mongohoused.log
-         remote_file: mongosql-odbc-driver/artifacts/logs/${version_id}/${build_variant}/mongohoused.log
-         bucket: mciuploads
-         permissions: public-read
-         content_type: application/octet-stream
-     - command: s3.put
-       params:
-         aws_key: ${aws_key}
-         aws_secret: ${aws_secret}
-         local_file: mongosql-odbc-driver/local_adf/logs/mongodb_test.log
-         remote_file: mongosql-odbc-driver/artifacts/logs/${version_id}/${build_variant}/mongodb_test.log
-         bucket: mciuploads
-         permissions: public-read
-         content_type: application/octet-stream
+    - command: s3.put
```

Comment:
Did your IDE do this? I'm fine with this sort of thing, but I know specific people who don't like to see noise like this in PRs about something else (lol) So just keep in mind when you have reviewers who aren't me that some people _really_ don't like this. I don't think @nbagnard cares either?

Reply:
Yeah it was my IDE doing some automatic formatting.

Reply:
I checked and I don't seem to have a default formatter set up for yaml files. I'll dig a bit to see what vscode applies by default.